Makefile

```diff
@@ -285,21 +285,17 @@ endef
 ##@ Bundle
 
 .PHONY: operator-sdk
-OSDK = ./bin/operator-sdk
+OSDK = $(shell pwd)/bin/operator-sdk
+OPERATOR_SDK_VERSION = v1.24.0
 operator-sdk: ## Download operator-sdk locally if necessary.
-ifeq (,$(wildcard $(OSDK)))
-ifeq (,$(shell which operator-sdk 2>/dev/null))
-	@{ \
+	@test '$(shell $(OSDK) version | cut -d" " -f3 | tr -d "," | tr -d '"')' = '$(OPERATOR_SDK_VERSION)' ||\
```

Comment:
This is very hard to read. It was bad before this change but now it is even worse.

Can we extract this code to external tool that check if a specific version of a tool exists, and download it if needed? The external tool can be written in a more manageable language that does not need such crazy and unclear syntax.

We need the same thing for all the tools installed by drenv, so probably we should keep this bad part as is for now and solve this issue once for all tools later.